<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addDialog</class>
 <widget class="QDialog" name="addDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>38</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>学号：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>姓名：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>成绩：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>名次：</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>111</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="idlineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="namelineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="scorelineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="ranklineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="calculatepushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>换算</string>
   </property>
  </widget>
  <widget class="QPushButton" name="yesButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="noButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="umbrellaLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>0</y>
     <width>141</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="icon.qrc">:/images/umbrella.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>noButton</sender>
   <signal>clicked()</signal>
   <receiver>addDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yesButton</sender>
   <signal>clicked()</signal>
   <receiver>addDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>74</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
